<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTable</class>
 <widget class="QDialog" name="DlgTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DlgTable</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>确认(&amp;O)</string>
       </property>
       <property name="icon">
        <iconset resource="plugin_modbus_rtu.qrc">
         <normaloff>:/images/ok</normaloff>:/images/ok</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>取消(&amp;C)</string>
       </property>
       <property name="icon">
        <iconset resource="plugin_modbus_rtu.qrc">
         <normaloff>:/images/cancel</normaloff>:/images/cancel</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="plugin_modbus_rtu.qrc"/>
 </resources>
 <connections/>
</ui>
